app/bundles/EmailBundle/Entity/EmailRepository.php

```diff
@@ -198,6 +198,7 @@ public function getEmailPendingLeads($emailId, $listIds = null, $countOnly = fal
                 ->setMaxResults($limit);
         }
 
+        $q->orderBy('l.id');
```

Comment:
This change causes following error on Postgres
```
 500 Internal Server Error - An exception occurred while executing 'SELECT count(l.id) as count FROM mau_leads l INNER JOIN mau_lead_lists_leads ll ON l.id = ll.lead_id INNER JOIN mau_email_list_xref el ON el.leadlist_id = ll.leadlist_id WHERE (el.email_id = 24) AND (l.id NOT IN (SELECT dne.lead_id FROM mau_email_donotemail dne WHERE dne.lead_id IS NOT NULL)) AND (l.id NOT IN (SELECT stat.lead_id FROM mau_email_stats stat WHERE (stat.lead_id IS NOT NULL) AND (stat.email_id = el.email_id))) AND ((l.email IS NOT NULL) OR (l.email <> '')) ORDER BY l.id ASC': SQLSTATE[42803]: Grouping error: 7 ERROR: column "l.id" must appear in the GROUP BY clause or be used in an aggregate function LINE 1: ...((l.email IS NOT NULL) OR (l.email <> '')) ORDER BY l.id ASC ^
```
If I add also `$q->groupBy('l.id');`, the query works fine.